testutils/metrics/metrics.go

```diff
@@ -35,9 +35,11 @@ type TestStatsReporter struct {
 	GaugesWG   sync.WaitGroup
 	TimersWG   sync.WaitGroup
 
-	Counters map[string]int64
-	Gauges   map[string]float64
-	Timers   map[string]time.Duration
+	Counters           map[string]int64
+	Gauges             map[string]float64
+	Timers             map[string]time.Duration
+	HistogramSamples   map[string]tally.Buckets
```

Comment:
Has this been implemented internally? If not we cannot use it until the internal implementation support these APIs.